deepmd/fit/ener.py

```diff
@@ -856,9 +856,12 @@ def change_energy_bias(
             delta_bias = np.linalg.lstsq(type_numbs, bias_diff, rcond=None)[0]
             unbias_e = energy_predict + type_numbs @ delta_bias
             atom_numbs = type_numbs.sum(-1)
-            rmse_ae = (
-                np.sqrt(np.square(unbias_e - energy_ground_truth)) / atom_numbs
-            ).mean()
+            rmse_ae = np.sqrt(
+                np.mean(
+                    np.square(unbias_e.ravel() - energy_ground_truth.ravel())
+                    / atom_numbs
+                )
+            )
```

Comment:
```suggestion
                    np.square(unbias_e.ravel() - energy_ground_truth.ravel())
                )
            ) / atom_numbs
```

Reply:
I disagree. `atom_numbs` is in shape `(N,)`, so it should be used before `np.mean`.

Reply:
The current revision normalize the total energy by sqrt(N), what i am suggesting is to normalize it by N. One may divide by `atom_numbs**2`.

Reply:
You may mean
```suggestion
                    np.square((unbias_e.ravel() - energy_ground_truth.ravel()) / atom_numbs)
                )
            )
```